components/menu/menu-item.js

```diff
@@ -15,22 +15,18 @@ class MenuItem extends MenuItemMixin(LitElement) {
 		return [ menuItemStyles,
 			css`
 				:host {
+					align-items: center;
 					display: flex;
 					padding: 0.75rem 1rem;
 				}
 
-				:host > span {
-					flex: auto;
-					line-height: 1rem;
-					overflow-x: hidden;
-					overflow-y: hidden;
-					text-overflow: ellipsis;
-					white-space: nowrap;
-				}
-
-				:host > d2l-icon {
+				d2l-icon {
 					flex: none;
-					margin-top: 0.1rem;
+					margin-left: 6px;
```

Comment:
I added `6px` of left margin to the ">" icon to make sure the supporting text (and the menu item text too) doesn't butt up against it. I verified this value with Jeff G.